backend/app/adapter/graphql/resolver/authmutation.go

```diff
@@ -5,14 +5,16 @@ import (
 	"time"
 
 	"github.com/short-d/short/app/entity"
+	"github.com/short-d/short/app/usecase/changelog"
 	"github.com/short-d/short/app/usecase/url"
 )
 
 // AuthMutation represents GraphQL mutation resolver that acts differently based
 // on the identify of the user
 type AuthMutation struct {
-	user       *entity.User
-	urlCreator url.Creator
+	user             *entity.User
+	changeLogCreator changelog.Creator
```

Comment:
```suggestion
	changeLog changelog.ChangeLog
```
Having separate creator and reader is over engineering. I am sorry for my tech debt for URL creator and reader. They will be merged in the future.